csrc/bn.h

```diff
@@ -139,6 +139,9 @@ class BigNumObj {
 
 inline BigNumObj bn_zero() { return BigNumObj(); }
 
+// Copies src to dst.
```

Comment:
This is a misleading comment. It does not simply copy from src to dst, it frees whatever is behind the dst pointer, allocates new memory for a copy of src, and returns that into the dst pointer.

Reply:
Updated the comment to reflect this.